pkgs/development/interpreters/bqn/cbqn/default.nix

```diff
@@ -95,6 +95,9 @@ stdenv.mkDerivation rec {
     install -Dm644 include/bqnffi.h -t "$dev/include"
     install -Dm755 libcbqn${stdenv.hostPlatform.extensions.sharedLibrary} -t "$lib/lib"
   ''
+  + lib.optionalString stdenv.hostPlatform.isDarwin ''
+    install_name_tool -id $lib/lib/libcbqn.dylib $lib/lib/libcbqn.dylib
```

Comment:
Should be possible to use fixDarwinDylibNames for this instead.

Reply:
I wasn't aware of `fixDarwinDylibNames`. Fixed and seems to work.